Cargo.toml

```diff
@@ -45,6 +46,9 @@ rayon = "1.2.1"
 humantime = "2.0.0"
 wasmparser = "0.71.0"
 
+[patch.crates-io]
+wiggle = { path = "crates/wiggle" }
```

Comment:
I'm a little worried about this myself, `wiggle` isn't super stable in its interface an APIs, and I think we'd like to reserve the right to change them at any time. If a bundled dependency in a submodule (e.g. the wasmtime integration in the wasi-crypto repository) uses `wiggle` as defined in-tree then we won't be able to change the one in-tree if it breaks wasi-crypto.

Would it be possible to have the wiggle integration live in this repository instead of the wasi-crypto repository? Ideally the integration in the wasi-crypto repository would only depend on non-wasmtime crates on crates.io so we can be sure that changes in wasmtime won't accidentally break it.

Reply:
The implementation has been moved to this repository. It indeed only depends on non-wasmtime crates beyond `wiggle`, so that should be fine.

Thank you!